static/filters/index.md

```diff
@@ -63,7 +63,7 @@ This example body can be sent via post to create a filter, with the response giv
 }
 ```
 
-Futher PUT requests can be made to `/filters/{filter-ID}` to update the dimensions required and once the request is complete it can be submitted by adding `?state=submitted`.
+For CMD datasets, futher PUT requests can be made to `/filters/{filter-ID}` to update the dimensions required and once the request is complete it can be submitted by adding `?state=submitted`.
```

Comment:
Can further PUT requests be made for both CMD and Census 2021 datasets? Is the difference only in the way we complete the journey (url parameter vs endpoint)? If so, it would make sense rewording this to make it clearer. At the moment it might look like only CMD datasets allows further (misspelled, by the way) requests.

Also, the parameter mentioned in this paragraph does not match the one in the examples: `state=submitted` vs `submitted=true`, which one is correct?

Reply:
I've now created a separate page to document how a Census 2021 filter is created, and removed the PUT information from the CMD page as the PUT endpoint does not work as expected.  I hope this is a bit clearer now.